test/linter/test-consistency.js

```diff
@@ -73,17 +74,41 @@ class ConsistencyChecker {
     }
 
     // Check sub-features.
-    const keys = Object.keys(data).filter((key) => key != '__compat');
-    keys.forEach((key) => {
+    this.getSubfeatures(data).forEach((key) => {
```

Comment:
FWIW Unrelated code smell: The `ConsistencyError` type (defined on lines 22-25) has an extraneous property `feature`, which is not added by `checkSubfeatures()` doesn't add. Given that `feature` (line 64) is directly inferred from `path`, that property might not be used anywhere.